tests/integration/targets/meraki_ssid/tasks/main.yml

```diff
@@ -440,122 +476,139 @@
     delegate_to: localhost
     register: set_radius_server_idempotent
 
-  - debug:
+  - name: Debug set_radius_server_idempotent
+    ansible.builtin.debug:
       var: set_radius_server_idempotent
 
-  - assert:
+  - name: Assert set_radius_server_idempotent
+    ansible.builtin.assert:
       that:
         - set_radius_server_idempotent is not changed
 
   #################
   # Error testing #
   #################
   - name: Set available on all aps to false without also setting availability tags
-    meraki_ssid:
-      auth_key: '{{auth_key}}'
+    cisco.meraki.meraki_mr_ssid:
+      auth_key: '{{ auth_key }}'
       state: present
-      org_name: '{{test_org_name}}'
+      org_name: '{{ test_org_name }}'
       net_name: TestNetSSID
       name: AnsibleSSID
       available_on_all_aps: false
     delegate_to: localhost
-    register: no_availability_tags
-    ignore_errors: yes
+    register: false _availability_tags
```

Comment:
This variable name is invalid and giving an error.

Reply:
Yup, I think that did my script I will fix it now ;)